### PR TITLE
Added Bjarne Stroustrup and Susanne Bødker

### PR DESCRIPTION
**Pull request by AndersBallegaard**
Added Bjarne Stroustrup to namegenerator

### CODE REVIEW

**Comment by thaJeztah** [comment]
Can you also add a female to the list, per: https://github.com/moby/moby/blob/06ca1606e1a09c36bd12ed503b4c99d8a17ca230/pkg/namesgenerator/names-generator.go#L121

**Review by thaJeztah — CHANGES_REQUESTED** [review]
(see my comment)

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -189,6 +189,9 @@ var (
 		// Elizabeth Blackwell - American doctor and first American woman to receive a medical degree - https://en.wikipedia.org/wiki/Elizabeth_Blackwell
 		"blackwell",
 
+		// Susanne Bødker - Researcher and author focued on human-computer interaction https://en.wikipedia.org/wiki/Susanne_B%C3%B8dker
+		"Bodker",
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go:193]: Can you make this lowercase?

```suggestion
		"bodker",
```

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks for updating; I left one more comment; can you address that (amending the existing commit); perhaps squash the commits as well.

**Comment by tianon** [comment]
Thank you so much for taking the time to make this contribution!  We're very sorry for how long it has taken us to come to a decision on it, but as of https://github.com/moby/moby/pull/43210 the `namesgenerator` package is now officially "frozen" against new additions.  There is some explanation there as to why it is frozen (and why it has taken us so long to come to a decision on PRs like yours 🙈), but https://github.com/moby/moby/pull/43210#issuecomment-1029934277 is especially good for a longer explanation. :heart:

**Pull request closed without merging**